hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OzoneClientKeyReadWriteOps.java

```diff
@@ -0,0 +1,247 @@
+          defaultValue = "100")
+  private int percentageRead;
+
+  @CommandLine.Option(names = {"--clients"},
+          description =
+                  "Number of clients, defaults 1.",
+          defaultValue = "1")
+  private int clientsCount = 1;
+
+  @CommandLine.Option(
+          names = "--om-service-id",
+          description = "OM Service ID"
+  )
+  private String omServiceID = null;
+
+  private Timer timer;
+
+  private OzoneBucket[] ozoneBuckets;
+
+  private byte[] keyContent;
+
+  private static final Logger LOG =
+          LoggerFactory.getLogger(OzoneClientKeyReadWriteOps.class);
+  public enum TaskType {
+    READ_TASK,
+    WRITE_TASK
+  }
+  private KeyGeneratorUtil kg;
+
+
+  @Override
+  public Void call() throws Exception {
+    init();
+    OzoneConfiguration ozoneConfiguration = createOzoneConfiguration();
+    OzoneClient[] ozoneClients = new OzoneClient[clientsCount];
+    for (int i = 0; i < clientsCount; i++) {
+      ozoneClients[i] = createOzoneClient(omServiceID, ozoneConfiguration);
+    }
+
+    ensureVolumeAndBucketExist(ozoneClients[0], volumeName, bucketName);
+    ozoneBuckets = new OzoneBucket[clientsCount];
+    for (int i = 0; i < clientsCount; i++) {
+      ozoneBuckets[i] = ozoneClients[i].getObjectStore().getVolume(volumeName)
+              .getBucket(bucketName);
+    }
+
+    timer = getMetrics().timer("key-read-write");
+    if (objectSizeInBytes >= 0) {
+      keyContent = RandomUtils.nextBytes(objectSizeInBytes);
+    }
+    if (kg == null) {
+      kg = new KeyGeneratorUtil();
+    }
+    runTests(this::readWriteKeys);
+
+    for (int i = 0; i < clientsCount; i++) {
+      if (ozoneClients[i] != null) {
+        ozoneClients[i].close();
+      }
+    }
+    return null;
+  }
+
+  public void readWriteKeys(long counter) throws Exception {
+    int clientIndex = (int)((counter) % clientsCount);
+    OzoneBucket ozoneBucket = ozoneBuckets[clientIndex];
+    TaskType taskType = decideReadOrWriteTask();
+    String keyName = getKeyName(taskType, clientIndex);
+
+    timer.time(() -> {
+      try {
+        switch (taskType) {
+          case READ_TASK:
+          processReadTasks(keyName, ozoneBucket);
+          break;
+          case WRITE_TASK:
+          processWriteTasks(keyName, ozoneBucket);
+          break;
+        default:
+          break;
+        }
+      } catch (RuntimeException ex) {
+        LOG.error(ex.getMessage());
+        throw ex;
+      } catch (IOException ex) {
+        LOG.error(ex.getMessage());
+        throw new RuntimeException(ex.getMessage());
+      }
+
+    });
+  }
+
+  public void processReadTasks(String keyName, OzoneBucket ozoneBucket)
+          throws RuntimeException, IOException {
+    if (readMetadataOnly) {
+        ozoneBucket.getKey(keyName);
+    } else {
+      byte[] data = new byte[objectSizeInBytes];
+      try (OzoneInputStream inputStream = ozoneBucket.readKey(keyName)) {
+        inputStream.read(data);
```

Comment:
@kerneltime  hey Ritesh, if I don't need to access the result from read method, is there a way we could suppress the build error thrown from git workflow by this line (because the build throws an error that we ignore the result of read method)? Thanks!

Reply:
You can add a `@SuppressWarnings({"unused"})` to tell findbugs make an exception here.